criteria.go

```diff
@@ -132,12 +132,16 @@ func (f *field) Neq(value interface{}) *Criteria {
 func (f *field) In(values ...interface{}) *Criteria {
 	return &Criteria{
 		p: func(doc *Document) bool {
-			normValues, err := encoding.Normalize(getFieldOrValue(doc, values))
-			if err != nil || normValues == nil {
-				return false
+			normValues := make([]interface{}, 0, len(values))
+			for _, v := range values {
+				normValue, err := encoding.Normalize(getFieldOrValue(doc, v))
+				if err != nil {
+					return false
```

Comment:
In this way, the criteria will always return false, even if there is a value which is normalized without error which is equal to the document field. So, I would write something like:

```golang
docValue := doc.Get(f.name)
for _, v := range values {
    normValue, err := encoding.Normalize(getFieldOrValue(doc, v))
    if err != nill && compareValues(normValue, docValue) == 0 {
	    return true
    }
}
```